File: Python/Types of Matrix/Column matrix.py
# Column matrix is that has 1 column and multiple rows
A = [[1],
     [2],
     [3]]

if len(A) > 1 and len(A[0]) == 1:
    print("Column matrix: ")
    for row in range(0, len(A)):
        for col in range(0, len(A[row])):
            print("\t", A[row][col], end='')
        print()
else:
    print("=> This isn't a column matrix.")

File: Java/Types of Matrix/Column matrix.java
class column_matrix {
    public static void main(String[] args) {
        // Column matrix is that has 1 column and multiple rows
        int[][] A = {
            { 1 },
            { 2 },
            { 3 }
        };
        
        if(A.length > 1 && A[0].length == 1){
            System.out.println("Column matrix: ");
            for (int row = 0; row < A.length; row++) {
                for (int col = 0; col < A[row].length; col++) {
                    System.out.print("\t" + A[row][col]);
                }
                System.out.println();
            }
        }else{
            System.out.println("=> This isn't a column matrix.");
        }
    }
}